scripts/serve.js

```diff
@@ -39,19 +39,16 @@ const packages = fs.readdirSync(packagesPath);
 
 const app = express();
 
-packages.forEach((name) => {
+packages.forEach(name => {
   const root = path.join(packagesPath, name);
   const entry = path.join(root, 'index.js');
   const dest = path.join(root, `${name}.bundle.js`);
-  const external = (id) => /^@(annotator|hot)/.test(id);
+  const external = id => /^@(annotator|hot)/.test(id);
```

Comment:
Is the removal of parens from them `google` presets? I actually rather like having them around as I find hunting for the extra `>` in a line tedious.

But again...mailing list? :wink: